softwarereview_intro.pt.Rmd

```diff
@@ -47,17 +47,17 @@ Além disso, os autores e revisores podem entrar em contato com os editores em p
 - O repositório do seu pacote será transferido para a organização rOpenSci.
```

Comment:
```suggestion
- Se você quiser, o repositório de seu pacote será transferido para a organização rOpenSci.
```

@beatrizmilz this is the change made in #864, could we please add it for Portuguese?